### PR TITLE
Add ToastNotificationManger::Setting API

### PR DESCRIPTION
**Pull request by pmpurifoy**
- Implements the ToastNotificationManager::Setting API
- Includes unit tests checking for the enabled scenario for unpackaged/packaged applications

### CODE REVIEW

File: dev/ToastNotifications/ToastNotificationManager.cpp [hunk 1]
```diff
@@ -96,7 +96,11 @@ namespace winrt::Microsoft::Windows::ToastNotifications::implementation
     }
     winrt::Microsoft::Windows::ToastNotifications::ToastNotificationSetting ToastNotificationManager::Setting()
     {
-        throw hresult_not_implemented();
+        std::wstring appId { RetrieveAppId() };
+        DWORD toastNotificationSetting{ 0 };
+        ToastNotifications_QuerySettings(appId.c_str(), 0, &toastNotificationSetting);
```
Comment (sharath2727) [inline dev/ToastNotifications/ToastNotificationManager.cpp:101]: add a comment why we are passing 0?

File: test/TestApps/ToastNotificationsTestApp/main.cpp [hunk 2]
```diff
@@ -186,6 +186,16 @@ bool VerifyFailedToastAssetsWithNullIconPath_Unpackaged()
     return false;
 }
 
+bool VerifyToastSettingEnabled_Unpackaged()
+{
+    return winrt::ToastNotificationManager::Default().Setting() == winrt::ToastNotificationSetting::Enabled;
+}
+
+bool VerifyToastSettingEnabled()
```
Comment (sharath2727) [inline test/TestApps/ToastNotificationsTestApp/main.cpp:194]: curious if we can reuse the same API for both packaged and unpackaged here?

**Review by sharath2727 — CHANGES_REQUESTED** [review]

**Review by danielayala94 — APPROVED** [review]
Make sure to have a task for the unit tests that verify the Disabled setting scenarios.

File: dev/ToastNotifications/ToastNotificationManager.cpp [hunk 3]
```diff
@@ -96,7 +96,11 @@ namespace winrt::Microsoft::Windows::ToastNotifications::implementation
     }
     winrt::Microsoft::Windows::ToastNotifications::ToastNotificationSetting ToastNotificationManager::Setting()
     {
-        throw hresult_not_implemented();
+        std::wstring appId { RetrieveAppId() };
+        DWORD toastNotificationSetting{ 0 };
+        ToastNotifications_QuerySettings(appId.c_str(), 0 /* ToastNotifications */, &toastNotificationSetting);
```
Comment (hulumane) [inline dev/ToastNotifications/ToastNotificationManager.cpp:101]: @sharath2727 Please get rid of the toastNotificationtype argument (second one) in the UDK. It is redundant.

**Review by hulumane — APPROVED** [review]

**Comment by danielayala94** [comment]
:shipit:

File: dev/ToastNotifications/ToastNotificationManager.cpp (hunk 3 above)
Comment (pmpurifoy) [reply dev/ToastNotifications/ToastNotificationManager.cpp:101]: This will be a different task to fix the UDK correct?

**Review by hulumane — APPROVED** [review]

**Pull request merged**